components/admin_console/admin_definition.jsx

```diff
@@ -5432,24 +5439,10 @@ const AdminDefinition = {
                         help_text_markdown: true,
                         isHidden: it.not(it.licensed), // E10 and higher
                         isDisabled: it.any(
-                            it.not(it.userHasWritePermissionOnResource('experimental')),
+                            it.not(it.userHasWritePermissionOnResource(RESOURCE_KEYS.EXPERIMENTAL.FEATURES)),
                             it.stateIsFalse('ThemeSettings.EnableThemeSelection'),
                         ),
                     },
-
-                    // {
-                    //     type: Constants.SettingsTypes.TYPE_LIST,
```

Comment:
Can we remove this section of commented code? 
Noticed it's been there since 2 years ago untouched...
@reviewers

Reply:
These were introduced in https://github.com/mattermost/mattermost-webapp/pull/2588 and haven't been modified or touched since. Doesn't feel right to leave commented code in the codebase for this long.

Reply:
Yeah take it out!